proto/zepben/protobuf/nc/nc-data.proto

```diff
@@ -187,6 +189,8 @@ message NetworkIdentifiedObject {
         zepben.protobuf.cim.extensions.iec61968.metering.PanDemandResponseFunction panDemandResponseFunction = 83;
         zepben.protobuf.cim.extensions.iec61970.base.wires.BatteryControl batteryControl = 84;
         zepben.protobuf.cim.iec61970.base.wires.PerLengthPhaseImpedance perLengthPhaseImpedance = 85;
+        zepben.protobuf.cim.iec61970.base.wires.Cut cut = 86;
```

Comment:
This is a non-breaking change, but the PR shouldn't be merged until the other PR's are ready due to exhaustive when checks.